plugins/source/azure/docs/tables/README.md

```diff
@@ -11,100 +11,29 @@
     - [azure_cdn_rules](azure_cdn_rules.md)
   - [azure_cdn_security_policies](azure_cdn_security_policies.md)
 - [azure_compute_disks](azure_compute_disks.md)
-- [azure_compute_virtual_machine_scale_sets](azure_compute_virtual_machine_scale_sets.md)
 - [azure_compute_virtual_machines](azure_compute_virtual_machines.md)
-  - [azure_compute_instance_views](azure_compute_instance_views.md)
-  - [azure_compute_virtual_machine_extensions](azure_compute_virtual_machine_extensions.md)
-- [azure_container_managed_clusters](azure_container_managed_clusters.md)
+  - [azure_compute_virtual_machine_scale_sets](azure_compute_virtual_machine_scale_sets.md)
```

Comment:
Looks like we're missing quite a few tables/resources, for example I can't see all the SQL ones.
Are we planning to add those or I'm missing something?

Reply:
done